test/expected/tablespace.out

```diff
@@ -224,6 +265,13 @@ SELECT * FROM show_tablespaces('tspace_2dim');
  tablespace2
 (2 rows)
 
+SELECT * FROM hypertable_tablespaces;
+ hypertable  | tablespace  
+-------------+-------------
+ tspace_1dim | 
```

Comment:
What does it mean to have a NULL tablespace column in this view? The information seems inconsistent with `show_tablespaces('tspace_1dim')` above, which shows `tablespace2` for `tspace_1dim`.

Reply:
A NULL table space just means that the default tablespace is set. See the comment that I added to the PR regarding how tablespaces are set and handled.